strawberry/__init__.py

```diff
@@ -14,6 +16,10 @@
 from .union import union
 
 
+if TYPE_CHECKING:
+    from . import django
+
+
 __all__ = [
```

Comment:
not sure if should change the all though (as it now includes django)

Reply:
Well the reason we have `__all__` is _for_ type checking

Reply:
Does it do anything at run time though?

Reply:
Afaik, the only runtime effect it has is on `from strawberry import *`

Reply:
But we could just dynamically add to `__all__`:
```python
if TYPE_CHECKING:
    __all__.append("django")
```

Reply:
Yeah, I was thinking that, but looks ugly :D and I’m not 100% sure type checker would support it, as they don’t run python. 

If it only affects star imports we are good I’d say :)